recipe/meta.yaml

```diff
@@ -1,22 +1,17 @@
-{% set version = "3.13.1" %}
+{% set version = "3.10.4" %}
 
 package:
   name: geos
   version: {{ version }}
 
 source:
   url: https://download.osgeo.org/geos/geos-{{ version }}.tar.bz2
-  sha256: df2c50503295f325e7c8d7b783aca8ba4773919cde984193850cf9e361dfd28c
+  sha256: d6fc11bcfd265cbf2714199174e4c3392d657551e5fd84c74c07c863b29357e3
   patches:
     - patches/0001-unconditional-geos-pc.patch
 
 build:
   number: 0
-  skip: true  # [win and vc<14]
-  run_exports:
-    # pretty poor backcompat.  SO name changes each release.
-    # https://abi-laboratory.pro/tracker/timeline/geos/
-    - {{ pin_subpackage('geos', max_pin='x.x.x') }}
```

Comment:
Am I correct to remove these for a build outside the `main` branch?

Reply:
No, I think the run exports should still stay.  The skip is probably not needed.